### PR TITLE
[WIP] Update: add htmlhint

### LINKED ISSUE #210: HTML linting
I think adding an HTML linter like [htmlhint-loader](https://github.com/mattlewis92/htmlhint-loader) would be helpful in the same way that eslint is helpful while developing. Thoughts?

### PR DESCRIPTION
**Pull request by Aladdin-ADD**
fixes #210

### CODE REVIEW

**Comment by MaxMilton** [comment]
Should we provide a `.htmlhintrc` file with some preset defaults too?

**Comment by MaxMilton** [comment]
Also, should this be an option together with ESLint or should it be a separate option during vue-cli init time? I'm not sure which is the better option to be honest. Thoughts anyone?

**Comment by nickmessing** [comment]
@Aladdin-ADD, @MaxMilton, I think having a separate option would be better than combining it with eslint since the whole vue ecosystem is non-opinionated and highly configurable, even tho I think nobody will use just one of them.

File: template/build/webpack.base.conf.js [hunk 1]
```diff
@@ -41,6 +41,12 @@ module.exports = {
       },
       {{/lint}}
       {
+        test: /\.html/,
+        enforce: 'pre',
+        loader: 'htmlhint-loader',
+        exclude: /node_modules/
+      },
```
Comment (nickmessing) [inline template/build/webpack.base.conf.js]: I think this should be wrapped in `{{#lint}}...{{/lint}}` just like `eslint-loader` or have another key in `meta.js` something like `htmlLint`

File: template/package.json [hunk 2]
```diff
@@ -37,6 +37,7 @@
     "eslint-friendly-formatter": "^2.0.7",
     "eslint-loader": "^1.7.1",
     "eslint-plugin-html": "^2.0.0",
+    "htmlhint-loader": "1.3.0",
```
Comment (nickmessing) [inline template/package.json]: Same as above comment

**Comment by nickmessing** [comment]
I think, just like @MaxMilton said, there should be `.htmlhintrc` and it would be nice to add `htmllint src/**/*.vue` to `scripts` in `package.json` I think.

**Comment by Aladdin-ADD** [comment]
I got an error running `npm run dev`, after adding this loader.
image

there are two issues here:
+ showing "Failed to compile with 1 errors", it is no much helpful to a user. can we change some config to show more info?
+ why this error? I just followed the htmlhint-loader's docs, did I forget something?

**Comment by MaxMilton** [comment]
Looks like there might soon be a solution for the blank error in `friendly-errors-webpack-plugin`, see this PR: https://github.com/geowarin/friendly-errors-webpack-plugin/pull/52

**Pull request closed without merging**